lib/testing/src/lib/process-services-cloud/pages/start-process-cloud-component.page.ts

```diff
@@ -50,6 +50,11 @@ export class StartProcessCloudPage {
         await BrowserActions.clearSendKeys(this.processNameInput, name);
     }
 
+    async getProcessName(): Promise<string> {
+        await BrowserVisibility.waitUntilElementIsVisible(this.processNameInput);
+        return await this.processNameInput.getAttribute('value');
```

Comment:
You don't need `return await` here
```suggestion
        return this.processNameInput.getAttribute('value');
```